src/AmbientSounds/ViewModels/ShellPageViewModel.cs

```diff
@@ -318,4 +318,9 @@ private void HandleTitleBanner()
             _telemetry.TrackEvent(TelemetryConstants.FocusTimeBannerClicked);
         }
     }
+
+    public async Task<bool> OpenConfirmCloseAsync()
+    {
+        return await _dialogService.OpenConfirmCloseAsync();
```

Comment:
Minor suggestion, but I don't think the async/await is necessary here since both methods return `Task<bool>`.

Reply:
It's probably better safe than sorry though, right?

Reply:
It could make debugging the method slightly harder but it's faster.

Reply:
Hmm, I'll take the faster route.